### ISSUE TITLE
Building and Landmark Icons

### ISSUE THREAD

**Issue by skesus** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#1355]
**Describe your suggestion in full detail below:**
I need building Icons. Take them from Warcraft 1, 2 or 3. Watch/Guard Towers, Scout Towers, Town Hall, Arcane Observatory/Sanctum, Orc Towers and Keeps, High Elven Towers, High Elven Cities, Brewery/Tavern (Thelsamar and Thunderbrew), Human Keeps (Shadowfang, Stonewatch, Fenris Isle, Bastion, Mardenholde, Durnholde), Dragon Roosts, Druid Buildings, Arcane Observatory,
Dwarf Forts (See Algaz Station), Dun Modr 

Basically I'd like to make Landmarks/Special Buildings and need things to resemble the above

**Comment by Raged50** [comment]
We are using original content where possible, I feel like this could be one of those circumstances
